src/compojure/api/meta.clj

```diff
@@ -189,7 +189,8 @@
       "    (created (path-for ::user {:id (random-int)}))))")))
 
 (defmethod restructure-param :name [_ v acc]
-  (update-in acc [:swagger] assoc :x-name v))
+  ;; :x-name could be possibly moved from [:info :public] into [:info]
```

Comment:
name is used in reverse routing, so it should definetely be on the top-level. Maybe push data to both? (usefull also for the clients)